web/Views/Property/_PropertyTable.cshtml

```diff
@@ -62,11 +62,11 @@
                 <td>
                     @if (prop.Availability == AvailabilityState.AvailableSoon && prop.AvailableFrom.HasValue)
                     {
-                        @("From " + prop.AvailableFrom.Value.ToShortDateString())
+                        @("Available" + "\n" + prop.AvailableFrom.Value.ToShortDateString())

```

Comment:
Should this say Available from instead of Available?

Reply:
I don't think it makes a massive difference - 'Available' followed by a date I think fairly clearly implies the 'from' part, but I can add it.